src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsKruxObject.java

```diff
@@ -55,16 +52,20 @@ public void verifyKruxControlTag(String kruxSiteId) {
   /**
    * Test whether the Krux user id is not empty and added to GPT calls
    */
-  public void verifyKruxUserParam() {
+  public void verifyKruxUserParam(String slotName) {
     String script = "return localStorage.kxuser;";
     waitForKrux();
     String user1 = (String) ((JavascriptExecutor) driver).executeScript(script);
     refreshPage();
     waitForKrux();
     String user2 = (String) ((JavascriptExecutor) driver).executeScript(script);
+    String gptPageParams = getGptPageParams(slotName);
     // TODO: figure out why we get krux user id in GPT calls from localStorage.kxuser in current PV OR from previous PV
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1135)